src/solvers.jl

```diff
@@ -111,6 +139,26 @@ function solve(m::Model; suppress_warnings=false,
                 fill(NaN, numRows)
             end
         end
+        # conic duals
+        if traits.conic && !traits.qp && !traits.qc && !traits.sdp
+            numBndRows = getBndRows(m)
+            numSOCRows = getSOCRows(m)
+            m.conicconstrDuals = try
+                MathProgBase.getconicdual(m.internalModel)
+            catch
+                fill(NaN, numRows+numBndRows+numSOCRows)
+            end
+            if m.conicconstrDuals[1] != NaN
```

Comment:
try and catch directly assigns to the m.conicconstrDuals, thats why I ended up moving that block outside of it, how can I move it inside? (right after getconicdual gives me an error, sorry for noobness!!)